app/models/manageiq/providers/openshift/container_manager/refresh_parser.rb

```diff
@@ -109,6 +120,50 @@ def parse_build_pod(build_pod)
                                                            build_pod.status.config.try(:name))
         new_result
       end
+
+      def parse_exposed_ports(exposed_ports)
+        exposed_ports.to_h.keys.each_with_object({}) do |port, h|
+          n, p = port.to_s.split('/')
+          h[n] = p
+        end
+      end
+
+      def parse_env_variables(env_variables)
+        env_variables.each_with_object({}) do |var_def, h|
+          name, value = var_def.split('=')
```

Comment:
@enoodle are you sure it's this simple? Also what if the value contains a `"="`?